pkgs/development/python-modules/cloudpathlib/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pytestCheckHook
+, flit-core
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "cloudpathlib";
+  version = "0.16.0";
+  pyproject = true;
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-zfzTXUbVKVh9dEFUoL35YqypU7clyHhM0uxHg1TqY6M=";
+  };
+
+  nativeBuildInputs = [ flit-core ];
+  propagatedBuildInputs = lib.optionals (pythonOlder "3.11") [ typing-extensions ];
+  # __main__.py: error: unrecognized arguments: --cov=cloudpathlib --cov-report=term --cov-report=html --cov-report=xml -n=auto
```

Comment:
Just remove the --cov arguments from latest.ini/setup.cfg

Reply:
The other PR already fixed this.